js/src/frameworks/openai.ts

```diff
@@ -76,11 +78,19 @@ export class OpenAIToolSet extends BaseComposioToolSet {
       file: this.fileName,
       params: { tool, entityId },
     });
+
+    const toolSchema = await this.getToolsSchema({
+      actions: [tool.function.name],
+    });
+    const appName = toolSchema[0]?.appName?.toLowerCase();
+    const connectedAccountId = appName && this.connectedAccountIds?.[appName];
+
     return JSON.stringify(
       await this.executeAction({
```

Comment:
Select the latest connectedAccountId for the app. We can filter it in the beginning also.

Reply:
- This has to be done if the connectedAccountIds is passed as empty object?